### ISSUE TITLE
Re: 'Deploy an emergency banner'

### ISSUE THREAD

**Issue by MalcolmVonMoJ** [alphagov/govuk-developer-docs#4552]
Problem with 'Deploy an emergency banner' (https://docs.publishing.service.gov.uk/manual/emergency-publishing.html)

The Local Emergency Banner doesn't seem to pass colour contrast checks.  I'm getting [an AA fail on WebAIM Colour contrast check](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=29A298).

**Comment by sengi** [comment]
Thanks for raising this!

I'm transferring the issue to [alphagov/govuk_publishing_components](https://github.com/alphagov/govuk_publishing_components/?tab=readme-ov-file#govuk-publishing-components) as that's what controls the [styling of the banner](https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/stylesheets/govuk_publishing_components/components/_emergency-banner.scss).

Someone from one of the frontend teams should be able to triage it from there.
